src/spyglass/utils/dj_helper_fn.py

```diff
@@ -559,6 +559,38 @@ def make_file_obj_id_unique(nwb_path: str):
     return new_id
 
 
+def _quick_get_analysis_path(file):
+    """Get the absolute path to an analysis file on disc without integrity checks.
+    For use when scanning large number of files.
+
+    Parameters
+    ----------
+    file : str
+        Name of the analysis file to get the path for.
+    Returns
+    -------
+    str
+        Absolute path to the analysis file. Returns None if the file is not found locally.
+    """
+    from spyglass.common import AnalysisNwbfile
+    from spyglass.settings import sg_config
+
+    analysis_dir = sg_config.analysis_dir
+    if os.path.exists(path := f"{analysis_dir}/{file}"):
```

Comment:
I can't remember if we were trying to use pathlib only or if mixing os.path and pathlib is okay

Reply:
I prefer pathlib. It requires a sep line for declaring the object, but then carries all sorts of methods like `exists` and `name` or `ext`